src/olympia/editors/templates/editors/review.html

```diff
@@ -29,128 +29,127 @@ <h2 class="addon"{{ addon.name|locale_html }}>
-  </h3>
-  <div id="review-files-paginate">
-    {% include "editors/includes/paginator_history.html"  %}
+  {% if user_changes %}
+  <div id="user-changes-history">
+    <h3>
+      {{ _('Add-on user change history') }}
+    </h3>
+    <ul id="user-changes">
+      {% for user_change in user_changes %}
+        <li>{{ user_change.activity_log.created }}: {{ user_change.activity_log }}</li>
+      {% endfor %}
+    </ul>
   </div>
-</div>
-<div class="results">
-  <div class="results-inner">
-    <table id="review-files" class="item-history">
-      {% for version in pager.object_list|reverse %}
-      <tr class="listing-header">
-        <th colspan="2">
-          {% trans version = version.version, created = version.created|datetime, version_status = version_status(addon, version) %}
-          Version {{ version }} &middot; {{ created }} <span class="light">&middot; {{ version_status }}</span>
-          {% endtrans %}
-        </th>
-      </tr>
-      <tr class="listing-body">
-        <td class="files">
-          {{ all_distinct_files(version) }}
-          {% if version.compatible_apps_ordered %}
-            <div><strong>{{ _('Compatibility:') }}</strong></div>
-            <ul>
-              {% for app, compat in version.compatible_apps_ordered %}
-              <li>
-                <div class="app-icon ed-sprite-{{ app.short }}" title="{{ app.pretty }}"></div>
-                {{ compat }}
-              </li>
-              {% endfor %}
-            </ul>
-          {% endif %}
-          {% if version.source %}
-              <div><strong>{{ _('Additional sources:') }}</strong></div>
-              <div>
-                  {% if is_admin %}
-                      <a href="{{ url('downloads.source', version.pk) }}">{{ _('Download files') }}</a>
-                  {% else %}
-                      {{ _('The developer has provided source code.')}}
-                  {% endif %}
-              </div>
-          {% endif %}
-        </td>
-        <td>
-          <table class="activity">
-            {% if version.releasenotes %}
-            <tr>
-              <th>{{ _('Version Notes') }}</th>
-              <td class="activity_version">
-                <div class="history-notes">
-                  {{ version.releasenotes|nl2br }}
-                </div>
-              </td>
-            </tr>
+  {% endif %}
+  <div id="review-files-header">
+    <h3 id="history">
+      {{ _('Add-on History') }}
+    </h3>
+    <div id="review-files-paginate">
+      {% include "editors/includes/paginator_history.html"  %}
+    </div>
+  </div>
+  <div class="results">
+    <div class="results-inner">
+      <table id="review-files" class="item-history">
+        {% for version in pager.object_list|reverse %}
+        <tr class="listing-header">
+          <th colspan="2">
+            <input type="checkbox" name="versions" value="{{ version.pk }}" autocomplete="off" class="data-toggle" data-value="reject_multiple_versions" />
```

Comment:
The checkbox is shown for all versions?  Regardless of if they were autoapproved?